fe/fe-core/src/main/java/com/starrocks/backup/RestoreJob.java

```diff
@@ -832,6 +869,30 @@ protected void sendCreateReplicaTasks() {
         }
     }
 
+    protected void addRestoreOlapView(List<View> restoredOlapViews) {
+        Database db = globalStateMgr.getLocalMetastore().getDb(dbId);
+
+        ConnectContext context = new ConnectContext();
```

Comment:
the context is not used

Reply:
`setThreadLocalInfo` will set the connectcontext as thread local varaible and will be used in `createView`

Reply:
Is those fileds enough for `createView()` ?

Reply:
Fixed, add sqlmode in connectContext